CPR/FaellesSQLCprServiceResult.php

```diff
@@ -21,51 +21,67 @@ public function __construct(array $serviceResult)
 
     private function setServiceResult(array $serviceResult)
     {
+        // The schema array describes which keys are expected to be present in the result and whether values has to be
```

Comment:
Consider using https://symfony.com/doc/current/components/options_resolver.html for this.